cli-plugins/apps/how-to-guides/how-to-guide-1.hbs.md

```diff
@@ -0,0 +1,28 @@
+# How-to-Guides simple reference
+
+## <a id='create-from-private-repo'></a> Create a workload from a private Git repository
+
+Check how to setup a workload from a Private Git repository in 
+[Out of the box supply chain with private Git repos](../../../scc/building-from-source.hbs.md#private-gitrepository),
+how the supply chain manages these type of workloads in
+[How it works](../../../scc/building-from-source.hbs.md#how-it-works) section and how to
+override parameters to customize the behavior to manage then in
+[Workload parameters](../../../scc/building-from-source.hbs.md#workload-parameters) section.
+
+## <a id='bind-service-to-workload'></a> Bind a service to a workload
+
+Tanzu Application Platform supports creating a workload with binding to multiple
+services ([ServiceBinding](../../service-bindings/about.hbs.md)).
```

Comment:
I think docs require links to be relative to the root instead of relative to the current file
```suggestion
services ([ServiceBinding](service-bindings/about.hbs.md)).
```
Please check for all the links in the page